modules/apps/knowledge-base/knowledge-base-api/src/main/java/com/liferay/knowledge/base/constants/KBArticleConstants.java

```diff
@@ -22,6 +22,8 @@
  */
 public class KBArticleConstants {
 
+	public static final long CLASS_NAME_ID_DEFAULT = 0;
+
```

Comment:
If I'm not mistaken, you've removed the usages of this constant. Can you remove the constant declaration too?